src/modules/chat_deleted_messages/index.js

```diff
@@ -33,10 +33,13 @@ class ChatDeletedMessagesModule {
   }
 
   handleMessage({message, preventDefault}) {
+    const clearChatBehavior = settings.get(SettingIds.CLEAR_BEHAVIOR);
```

Comment:
`handleMessage` gets called on every new message, may be more efficient to do this setting lookup inside the case statement to avoid unnecessary computation.

Reply:
eslint isn't happy with declarations in case statements, I can move the lookup into the if statement though since the variable isn't actually used again